package.json

```diff
@@ -51,7 +54,9 @@
   },
   "scripts": {
     "test": "mocha",
-    "compile": "babel src --out-dir dist",
+    "compile:cjs": "babel src --out-dir dist",
+    "compile:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "compile": "npm-run-all --parallel compile:cjs compile:es",
```

Comment:
It doesn't take long at all--I just like `npm-run-all` because if you do `yarn compile`, it'll also run its arguments with `yarn` :)
I'd be happy to take it out if you'd prefer not taking on an extra dependency, we don't gain much from it.